packages/accordions/src/elements/accordion/components/Label.tsx

```diff
@@ -9,6 +9,9 @@ import React, { forwardRef, HTMLAttributes } from 'react';
 import { StyledButton } from '../../../styled';
 import { useAccordionContext, useHeaderContext, useSectionContext } from '../../../utils';
 
+/**
+ * @extends HTMLButtonElement
```

Comment:
Should be `HTMLAttributes<HTMLButtonElement>`

Reply:
With this being a button the more specific type may need to be `HTMLButtonAttributes<HTMLButtonElement>`

Reply:
Fixed with Austin's type suggestion

Reply:
extends have been removed for now.